src/main/java/bfk/brickbreaker/GeneticLearn.java

```diff
@@ -0,0 +1,103 @@
+package bfk.brickbreaker;
+
+import java.util.*;
+import basicneuralnetwork.*;
+import static bfk.brickbreaker.BBFrame.NUM_BRICKS;
+
+public class GeneticLearn {
+
+    private static final int GENERATIONS = 10;
+    private static final int NETWORK_COUNT = 1000;
+    private static final int INPUT_SIZE = 2;
+    private static final int OUTPUT_SIZE = 2;
+    private static final int TOP_AMOUNT = (int) (NETWORK_COUNT * .10);
+    public static final int HIDDEN_NODES = 128;
+
+    public static void main(String[] args) {
```

Comment:
How long does it take to run this?

Reply:
Every generation takes about 30 minutes, is there a way to lessen that?

Reply:
Well you don't need a Frame